packages/cipherstash-proxy-integration/src/map_match_index.rs

```diff
@@ -25,5 +25,16 @@ mod tests {
             let result: String = row.get("encrypted_text");
             assert_eq!(encrypted_text, result);
         }
+
+        // TODO: See https://linear.app/cipherstash/issue/CIP-1337/operator-triggers-unmappable-error
+        // let sql = "SELECT id, encrypted_text FROM encrypted WHERE encrypted_text ~~ $1";
+        // let rows = client.query(sql, &[&"hello@"]).await.unwrap();
+
+        // assert_eq!(rows.len(), 1);
+
+        // for row in rows {
+        //     let result: String = row.get("encrypted_text");
+        //     assert_eq!(encrypted_text, result);
+        // }
     }
```

Comment:
I'd remove the commented out code - it's a public repo and the comment contains a link to our private issue tracker.

Reply:
Yes, good point